examples/oneview_enclosure.yml

```diff
@@ -18,11 +18,13 @@
   vars:
     config: "{{ playbook_dir }}/oneview_config.json"
     enclosure_name: 0000A66101
-    variant: C7000
+    enclosure_hostname: host_name
```

Comment:
not able to find host_name and ipv6 variables

Reply:
host_name is nothing but enclosure_hostname that's already defined in the config file and ipv6 variable is added to the config file.